pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel"
```

Comment:
Adding `wheel` was never necessary, and setuptools docs were fixed not to do that anymore recently.